src/libmongoc/src/mongoc/mongoc-apm.h

```diff
@@ -102,6 +102,9 @@ mongoc_apm_command_started_get_server_id (
 MONGOC_EXPORT (const bson_oid_t *)
 mongoc_apm_command_started_get_service_id (
    const mongoc_apm_command_started_t *event);
+MONGOC_EXPORT (int32_t)
+mongoc_apm_command_started_get_server_connection_id (
+   const mongoc_apm_command_started_t *event);
```

Comment:
Please add documentation pages to src/libmongoc/doc for the three new functions.